azurerm/internal/services/web/resourceids.go

```diff
@@ -0,0 +1,3 @@
+package web
+
+//go:generate go run ../../tools/generator-resource-id/main.go -path=./ -name=ManagedCertificate -id=/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/resGroup1/providers/Microsoft.Web/certificates/customhost.contoso.com
```

Comment:
_minor_ with the other PR's I've intentionally avoided using `0`'s for this - since this could indicate a parsing bug (whilst we don't check it at this point, these are UUID's and this is the default value for those) - it may be worth adding one in here:

```suggestion
//go:generate go run ../../tools/generator-resource-id/main.go -path=./ -name=ManagedCertificate -id=/subscriptions/12345678-1234-9876-4563-123456789012/resourceGroups/resGroup1/providers/Microsoft.Web/certificates/customhost.contoso.com
```